app/services/queries/cclow/litigation_query.rb

```diff
@@ -43,6 +44,12 @@ def filter_by_region
         scope.includes(:geography).where(geographies: {region: params[:region]})
       end
 
+      def filter_by_sector
+        return scope unless params[:law_sector].present?
+
+        scope.includes(:laws_sectors).where(laws_sectors: {id: params[:law_sector]})
```

Comment:
we can just use `joins` here, if we don't need to include the laws_sectors objects in Ruby.